src/App.js

```diff
@@ -1,10 +1,48 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import styled from 'styled-components';
 
+var axios = require('axios');
+
+const button = styled.button`
+  :focus{
+    outline:none;
+  }
+`
 function App() {
+  const [data, setData] = useState([]);
+
+  const loadCandidate = async () =>{
+    //await - 20번째 줄이 끝나야 그 다음 수행 (get을 기다리겠다): 비동기화
+    //result가 success라고 가정함 - 이때 result = response
+    const result = await axios.get('http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/candidates')
+    setData(result.data);
+    //14-19는 클릭시 -> 로 이동, 여기 있으면 새로고침 할 때마다 선종오빠한테 투표됨
+  }
+  async function clickVote(key, voteCount){
+    const config = {
+      params : {
+        id: key, //id가 0번인 사람을 투표해주세요
+        voteCount: voteCount + 1
+      }
+    }
+    //투표할 때 어떻게 한 표씩 증가시킬거냐- 화면 갱신
+    const vote = await axios.get('http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/vote?id=${id}', config)
```

Comment:
```suggestion
    const vote = await axios.get('http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/vote', config)
```
제가 찾아봤는데 config를 사용하시려면 이런식으로 사용하셔야 될거 같아요